exercises/practice/promises/.docs/introduction.md

```diff
@@ -0,0 +1,25 @@
+# Introduction
```

Comment:
I'm not sure how this file is displayed on the website, will it be visible by default before the instructions?

Reply:
I am pretty sure it will look the same as all the concept exercises where you first see the introduction text and then the instructions.